modules/mobile/app/services/mobile/v2/appointments/proxy.rb

```diff
@@ -6,12 +6,18 @@ module Mobile
   module V2
     module Appointments
       class Proxy
+        VAOS_STATUSES = %w[proposed cancelled booked fulfilled].freeze
+
         def initialize(user)
           @user = user
         end
 
-        def get_appointments(start_date:, end_date:, pagination_params:)
-          response = vaos_v2_appointments_service.get_appointments(start_date, end_date, nil, pagination_params)
+        def get_appointments(start_date:, end_date:, include_pending:, pagination_params:)
+          statuses = VAOS_STATUSES.dup
```

Comment:
had to do a shallow copy of the array to delete `proposed`. feels a bit awkward but seems worse to not freeze it in the first place.

Reply:
lol should've thought of that. Thanks.